recognition/GCN_S4654830/GCN-test-script.py

```diff
@@ -0,0 +1,20 @@
+from Data_prepare import *
```

Comment:
Module named GCN_Data pre.py (with a space), cannot be imported!

Reply:
Any chance to fix my error? or the mark already done